manifests/init.pp

```diff
@@ -28,6 +28,12 @@
 #   Default: true
 #   Run the system service on boot.
 #
+# [*service_initd_ensure*]
+#   Default for 'Ubuntu 15.04, 15.10, 16.04, 16.10' or 'Debian 8': "absent"
```

Comment:
This probably needs changing to "default for systemd systems, as determined by the $:: gitlab_systemd fact